build/scripts/DevEnv/NewDevEnv.ps1

```diff
@@ -0,0 +1,234 @@
+using module .\AppProjectInfo.class.psm1
+using module .\ALGoProjectInfo.class.psm1
+
+<#
+Creates a new dev env.
+#>
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingPlainTextForPassword', '', Justification = 'local build')]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '', Justification = 'local build')]
+[CmdletBinding(DefaultParameterSetName = 'ProjectPaths')]
+param(
+    [Parameter(Mandatory = $false)]
+    [string] $containerName = "BC-$(Get-Date -Format 'yyyyMMdd')",
+
+    [Parameter(Mandatory = $false)]
+    [string] $userName = 'admin',
+
+    [Parameter(Mandatory = $false)]
+    [string] $password = 'P@ssword1',
```

Comment:
I don't think we should default password parameters. If something need to be protected by a password, I'd suggest making the user provide it.

Reply:
Of course. I only placed it there because, when testing, I keep forgetting what password I used (or simply mistyped it).

Reply:
Fixed.